import_export/admin.py

```diff
@@ -833,6 +833,16 @@ def export_action(self, request, *args, **kwargs):
         context["title"] = _("Export")
         context["form"] = form
         context["opts"] = self.model._meta
+        context["fields_list"] = [
```

Comment:
Is it possible you could add a test which proves that this runs correctly

Reply:
done